### PR TITLE
Add CorrectionDesign

### PR DESCRIPTION
**Pull request by abeddow91**
## What does this change?
Adds the new design type for `Corrections and Clarifications`, `CorrectionDesign`.

### CODE REVIEW

File: client/src/main/scala/com.gu.contentapi.client/utils/CapiModelEnrichment.scala [hunk 1]
```diff
@@ -86,6 +86,7 @@ object CapiModelEnrichment {
         tagExistsWithId("tone/analysis") -> AnalysisDesign,
         tagExistsWithId("tone/comment") -> CommentDesign,
         tagExistsWithId("tone/letters") -> LetterDesign,
+        tagExistsWithId("theguardian/series/correctionsandclarifications") -> CorrectionDesign,
```
Comment (buck06191) [inline client/src/main/scala/com.gu.contentapi.client/utils/CapiModelEnrichment.scala:89]: Quick check on this: any chance we'll ever have this as well as other design flags e.g. letter or comment? If so it'll need to go higher up and have a test to check the edge cases

**Review by buck06191 — APPROVED** [review]
Approved with a query

File: client/src/main/scala/com.gu.contentapi.client/utils/CapiModelEnrichment.scala (hunk 1 above)
Comment (abeddow91) [reply client/src/main/scala/com.gu.contentapi.client/utils/CapiModelEnrichment.scala:89]: From what I've seen, there aren't any other flags for this.
